core/src/main/java/org/infinispan/transaction/xa/XaTransactionTable.java

```diff
@@ -180,20 +199,21 @@ private void forgetSuccessfullyCompletedTransaction(RecoveryManager recoveryMana
          LocalXaTransaction localTransaction, boolean committedInOnePhase) {
       final GlobalTransaction gtx = localTransaction.getGlobalTransaction();
       if (isRecoveryEnabled()) {
-         recoveryManager.removeRecoveryInformation(localTransaction.getRemoteLocksAcquired(), xid, false, gtx,
+         // TODO: this should call a different method that doesn't receive an ack
+         recoveryManager.removeRecoveryInformation(localTransaction.getRemoteLocksAcquired(), xid, gtx,
                partitionHandlingManager.isTransactionPartiallyCommitted(gtx));
          removeLocalTransaction(localTransaction);
       } else {
          releaseLocksForCompletedTransaction(localTransaction, committedInOnePhase);
       }
    }
 
-   private LocalXaTransaction getLocalTransactionAndValidate(Xid xid) throws XAException {
+   private LocalXaTransaction getLocalTransactionAndValidate(Xid xid) {
```

Comment:
The name is outdated, there's no validation, so I would suggest inlining the method.

Reply:
Sure.